CoreData/VICoreDataManager/VIManagedObjectMapperSKZ.m

```diff
@@ -25,11 +27,39 @@ @implementation VIManagedObjectMapperSKZ
 + (instancetype)mapperWithUniqueKey:(NSString *)comparisonKey andMaps:(NSArray *)mapsArray;
 {
     VIManagedObjectMapperSKZ *mapper = [[self alloc] init];
+    NSUInteger mapCount = mapsArray.count;
+    
+    id maps[mapCount];
+    id inputKeyPaths[mapCount];
+    id coreDataKeys[mapCount];
+    NSUInteger i = 0;
+    
+    for ( VIManagedObjectMapSKZ *map in mapsArray )
```

Comment:
I'd probably use `mapsArray enumerateObjectsUsingBlock: ...`, but you can't with the c style array. :+1: